### PR TITLE
Add a number of repos to k/community

### PR DESCRIPTION
**Pull request by cblecker**
fixes kubernetes/org#38.
fixes kubernetes/org#208.
fixes kubernetes/org#195.
fixes kubernetes/org#213.
fixes kubernetes/org#244.
fixes kubernetes/org#284.
fixes kubernetes/org#285.

/assign @spiffxp

### CODE REVIEW

File: sig-api-machinery/README.md [hunk 1]
```diff
@@ -92,6 +92,9 @@ The following subprojects are owned by sig-api-machinery:
     - https://raw.githubusercontent.com/kubernetes-incubator/client-python/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes/client-go/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/staging/src/k8s.io/client-go/OWNERS
+- **yaml**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-sigs/yaml/master/OWNERS
```
Comment (dims) [inline sig-api-machinery/README.md]: Thanks @cblecker !!

File: sig-architecture/README.md [hunk 2]
```diff
@@ -58,6 +58,9 @@ The following subprojects are owned by sig-architecture:
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/vendor/OWNERS
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/third_party/OWNERS
     - https://raw.githubusercontent.com/kubernetes/kubernetes/master/staging/OWNERS
+- **klog**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/klog/master/OWNERS
```
Comment (dims) [inline sig-architecture/README.md]: Thanks @cblecker !

**Comment by dims** [comment]
/lgtm
/hold

@cblecker holding for some more acks for other repos. please unhold when appropriate.

**Comment by cblecker** [comment]
I'm going to get this merged in, and then we can iterate.

/hold

**Comment by cblecker** [comment]
I'm going to get this merged in, and then we can iterate.

/hold cancel

**Pull request merged**